### ISSUE TITLE
bad results !

### ISSUE THREAD

**Issue by tianfengyijiu** [xuannianz/EfficientDet#51]
This is my command:
(base) E:\EfficientDet>python train.py --snapshot imagenet --phi 1 --gpu 0 --ran
dom-transform --compute-val-loss --freeze-backbone --batch-size 2 --steps 50000
pascal datasets\VOC2007
my dataset has 40 classes with 110000+ images.and after 45 epoches ,i get:

![image](https://user-images.githubusercontent.com/5301100/71437789-16a34880-272e-11ea-8f45-bb584fd94b01.png)
and mAP is 0.18,it is so bad.
what problem happen?

**Comment by jzq6520** [comment]
你是不是 两个阶段训练下来的，我试过直接训练效果也不好

**Comment by xuannianz** [comment]
@tianfengyijiu when batch_size is smaller than 16, you have to add --freeze-bn option.

**Comment by tianfengyijiu** [comment]
@xuannianz should I use --freeze-bn and --freeze-backbone at the same time?

**Comment by SiBensberg** [comment]
@xuannianz 
did using both work?
